quiz.js

```diff
@@ -0,0 +1,40 @@
+
+///////////////////////////// SCRATCH 3 /////////////////////////////
+
+var button = document.querySelector('#form input[type=submit]');
+var lines;
+var spaces;
+var spaceCount = 0;
+var enterKey = document.querySelector('#button');
+enterKey.addEventListener('keypress', function (evt) {
```

Comment:
So this to me looks like you have a keypress event on a button. Does this eventlistener log anything when it runs?